server/.env_example

```diff
@@ -15,4 +15,5 @@ DISTRIBUTION_DOMAIN=https://hmzsbpcw4ex33jxeiage.cloudfront.net
 CLOUD_FRONT_KEYPAIR_ID=Q8A4T7ZMUYRW3Z
 CLOUD_FRONT_PRIVATE_KEY=-----BEGIN RSA PRIVATE KEY-----RSA KEY FOR CLOUDFRONT=-----END RSA PRIVATE KEY-----
 KEY=s3_bucket_folder_name_can_customise_as_per_your_likings
-MONGO_URL=mongodb://localhost:27017/logoexecutive
\ No newline at end of file
+MONGO_URL=mongodb://localhost:27017/logoexecutive
+NEW_RELIC_LOG_ENABLED=false
```

Comment:
Change to ENV

Reply:
Added validation for all the environment variables.